math/ipopt/src/IpoptMinimizer.cxx

```diff
@@ -0,0 +1,256 @@
+#include <Math/IpoptMinimizer.h>
+#include <Fit/ParameterSettings.h>
+#include <Math/IFunction.h>
+#include "Math/MultiNumGradFunction.h"
+#include "Math/FitMethodFunction.h"
+#include <TString.h>
+
+using namespace ROOT;
+using namespace ROOT::Math;
+using namespace ROOT::Math::Experimental;
+using namespace Ipopt;
+using namespace ROOT::Fit;
+//_______________________________________________________________________
```

Comment:
Please consider adding doxygen documentation for the code you are adding, so that it appears in https://root.cern/guides/reference-guide.